cypress/cypress.config.js

```diff
@@ -18,7 +18,9 @@ module.exports = defineConfig({
   env: {
     OC_IDP: "kube:admin",
     CLUSTERSET: "submariner",
-    SUBMARINER_IPSEC_NATT_PORT: "4505"
+    SUBMARINER_IPSEC_NATT_PORT: "4505",
+    SUBMARINER_CATALOG_SOURCE: "submariner-catalog",
+    CLUSTER_LIST: ""
```

Comment:
Please, change the name of the variable to - MANAGED_CLUSTERS.
This is the variable we are using in our flow.